### PR TITLE
Support to run marvin in docker containers with irc-server and irc-client.

### PR DESCRIPTION
**Pull request by mosbth**
Add docker setup to run irc-server ngircd and irc-client irssi and a container for marvin to ease testing locally. Documented in the README on how to get going.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,6 +49,55 @@ A html report of the code coverage is generated into `build/coverage/index.html`
...
+Execute marvin in docker
+--------------------------
+
+Start by creating your own copy of the configuration file.
+
+```bash
+cp marvin_config_default.json marvin_config.json
+```
+
+Edit the following settings in the file `marvin_config.json`.
```
Comment (kh31d4r) [inline README.md:61]: if you pass the config to marvin on commandline instead, you can do it in the compose file and get rid of this extra step

File: docker-compose.yml [hunk 2]
```diff
@@ -1,7 +1,37 @@
...
 services:
+    ngircd:
+        image: lscr.io/linuxserver/ngircd:latest
+        container_name: ngircd
+        environment:
+            - PUID=1000
+            - PGID=1000
+            - TZ=Etc/UTC
+        volumes:
+            - ./.docker/ngircd/config:/config
```
Comment (kh31d4r) [inline docker-compose.yml:11]: do we really need this? I ran it just fine with the default config

Comment (mosbth) [reply docker-compose.yml:11]: I actually do not know. I tested without it first, got into issues, added the config, and so on. But you might be correct. I'll check it again.

File: README.md (hunk 1 above)
Comment (mosbth) [reply README.md:61]: That is true and ease the usage, I fix it.

**Comment by mosbth** [comment]
Fixed comments in #69

**Pull request merged**